collectors/python.d.plugin/ipfs/ipfs.chart.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, configuration=None, name=None):
         self.order = ORDER
         self.definitions = CHARTS
         self.baseurl = self.configuration.get('url', 'http://localhost:5001')
+        self.method = "POST"
```

Comment:
Well, it breaks compatibility with `ipfs` before `0.5.0`

It seems we need to check version first and then to set method. Is there an enpdoint th get version?

Reply:
Jus realized. A quick fix is to add 2nd job to the module config

https://github.com/netdata/netdata/blob/dc3f425a7d2992a8ea3df2c15622e032a37bf512/collectors/python.d.plugin/ipfs/ipfs.conf#L74-L77

=>

```yaml
localhost:
  name   : 'local'
  url    : 'http://localhost:5001'
  pinapi : no

localhost_v5:
  name   : 'local'
  url    : 'http://localhost:5001'
  method : POST
  pinapi : no
```

Reply:
where can i read about those issues? can you share the link?

Reply:
@ilyam8 yeah sure, the improvements are mentioned in the [blog](https://blog.ipfs.io/2020-04-28-go-ipfs-0-5-0/). But there was a massive amount of fixes that are not fully mentioned.

Running 0.4.x is highly discouraged, because it actually slows down the whole network.

Reply:
@ilyam8 here's the full changelog:

https://github.com/ipfs/go-ipfs/releases/tag/v0.5.0